inventory/byo/hosts.origin.example

```diff
@@ -113,6 +113,9 @@ openshift_release=v3.6
 # Downgrades are not supported and will error out. Be careful when upgrading docker from < 1.10 to > 1.10.
 # docker_version="1.12.1"
 
+# Specify whether to --enable-selinux on Docker (set false if using overlay/2 storage drivers before RHEL 7.4)
```

Comment:
Aha, yes, I did not understand it applied to both.

Better say "overlay / overlay2", then.